docs/source/llm_quantization/usage_guides/quantization.mdx

```diff
@@ -76,7 +76,7 @@ quantized_model = load_quantized_model(empty_model, save_folder=save_folder, dev
 
 ### Exllama kernels for faster inference
 
-For 4-bit model, you can use the exllama kernels in order to a faster inference speed. It is activated by default. If you want to change its value, you just need to pass `disable_exllama` in [`~optimum.gptq.load_quantized_model`]. In order to use these kernels, you need to have the entire model on gpus.
+For 4-bit model, you can use the exllama kernels in order to a faster inference speed. If you want to change its value, you just need to pass `disable_exllama` in [`~optimum.gptq.load_quantized_model`]. In order to use these kernels, you need to have the entire model on gpus.
```

Comment:
Sorry being picky: it could make sense to first give the example with exllamav2 (aka no need to pass any arg), and then have a small section explaining how to use the previous kernels from exllama. Here it is a bit weird having the least important part first.

Reply:
Makes sense !